use-cases/rag-on-gke/alloyDB/init-database-objects/init_database.tf

```diff
@@ -0,0 +1,153 @@
+data "google_client_config" "provider" {}
+
+data "google_container_cluster" "my_cluster" {
+  name     = var.gke_cluster_name
+  location = var.gke_cluster_location
+  project  = var.project_id
+}
+
+provider "kubernetes" {
+  host  = "https://${data.google_container_cluster.my_cluster.endpoint}"
```

Comment:
Have you tested this on the Playground as I don't believe this works with Connect Gateway.